reporter-web-app/config/webpack.config.js

```diff
@@ -14,7 +14,6 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const safePostCssParser = require('postcss-safe-parser');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
```

Comment:
Commit message:
* No need to repeat "plugin" in the title.
* Use present tense in the body, "Removing plugin..." -> " Remove the plugin..."

Reply:
Fixed.

Reply:
The body still says "Removing plugin..."

Reply:
Fixed